### ISSUE TITLE
Microbit radio

### ISSUE THREAD

**Issue by crisgplj** [lancaster-university/codal-microbit-v2#306]
Hi! This code is correct? https://lancaster-university.github.io/microbit-docs/ubit/radioevent/#listen
I cant use the radio with this.

I am interesting to manage radio without ubit.init() any idea?
Thanks!

**Comment by microbit-carlos** [comment]
Yes, that is also implemented in CODAL, have you tried it already? Are there any issues when it's running?

**Comment by crisgplj** [comment]
Hi! if you copy and paste the code of the page, you dont receive messages. I dont know if the problem is for me, because i use CODAL and maybe this code is for DAL.
